src/main/scala/nl/biopet/utils/tool/ToolDocumentation.scala

```diff
@@ -0,0 +1,163 @@
+package nl.biopet.utils.tool
+
+import java.io.{File, PrintWriter}
+
+import scala.io.Source
+
+trait ToolDocumentation {
```

Comment:
There are 2 options here:
- Extend this from ToolCommand
- Merge this inside ToolCommand so we have only 1 trait